demo/run-osm-demo.sh

```diff
@@ -14,7 +14,7 @@ fi
 source .env
 
 # Set meaningful defaults for env vars we expect from .env
-CI="${CI:-false}"  # This is set to true by Github Actions
+CI="${CI:-false}" # This is set to true by Github Actions
```

Comment:
Hey @C123R, would you mind carving out changes unrelated to this PR, such as this one in a separate PR instead. It would make it easier to review this PR. This includes other changes to indentation, as well.

Reply:
Sure, will do so.